Index: transcriber-ag/libs/aglib-2.0.1/src/ag/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/libs/aglib-2.0.1/src/ag/CMakeLists.txt	2012-01-21 23:00:01.000000000 +0100
+++ transcriber-ag/libs/aglib-2.0.1/src/ag/CMakeLists.txt	2012-02-06 00:27:58.000000000 +0100
@@ -50,3 +50,10 @@
 	
 	dl
 )
+
+install(
+	TARGETS ag
+
+	RUNTIME	DESTINATION bin
+	LIBRARY	DESTINATION lib
+)
\ No newline at end of file
Index: transcriber-ag/source/src/GUI/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/source/src/GUI/CMakeLists.txt	2012-01-21 23:00:01.000000000 +0100
+++ transcriber-ag/source/src/GUI/CMakeLists.txt	2012-02-06 00:27:58.000000000 +0100
@@ -167,3 +167,10 @@
 	${APPLE_LINKS}
 	${WIN32_LINKS}
 )
+
+install(
+	TARGETS ${ERR_APP} ${APP_NAME}
+
+	RUNTIME	DESTINATION bin
+	LIBRARY	DESTINATION lib
+)
\ No newline at end of file
Index: transcriber-ag/source/src/Formats/CTM/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/source/src/Formats/CTM/CMakeLists.txt	2012-01-21 23:00:01.000000000 +0100
+++ transcriber-ag/source/src/Formats/CTM/CMakeLists.txt	2012-02-06 00:27:58.000000000 +0100
@@ -82,7 +82,7 @@
 )
 
 install(
-	TARGETS	agfio_plugin_CTM ctm2tag 
+	TARGETS	agfio_plugin_CTM ctm2tag testCTM
 
 	RUNTIME	DESTINATION bin
 	LIBRARY	DESTINATION	lib/ag
Index: transcriber-ag/source/src/Formats/TransAG_compat/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/source/src/Formats/TransAG_compat/CMakeLists.txt	2012-01-21 23:00:01.000000000 +0100
+++ transcriber-ag/source/src/Formats/TransAG_compat/CMakeLists.txt	2012-02-06 00:27:58.000000000 +0100
@@ -102,7 +102,7 @@
 )
 
 install(
-	TARGETS	agfio_plugin_TransAG_compat testTransAG_compat
+	TARGETS	agfio_plugin_TransAG_compat testTransAG_compat compat2tag
 	
 	RUNTIME	DESTINATION bin
 	LIBRARY	DESTINATION	lib/ag
Index: transcriber-ag/source/src/MediaComponent/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/source/src/MediaComponent/CMakeLists.txt	2012-01-21 23:00:01.000000000 +0100
+++ transcriber-ag/source/src/MediaComponent/CMakeLists.txt	2012-02-06 00:27:58.000000000 +0100
@@ -275,5 +275,11 @@
 	${APPLE_LINKS}
 	${WIN32_LINKS}
 )
+install(
+	TARGETS	GenPeaks MMPlayer MediumGuesser MultimediaRTSPServer ThumbnailerTest audioTool
+	
+	RUNTIME	DESTINATION bin
+	LIBRARY	DESTINATION lib
+)
 endif (VIDEO_BUILD)
 
Index: transcriber-ag/source/src/Tools/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/source/src/Tools/CMakeLists.txt	2012-01-21 23:00:01.000000000 +0100
+++ transcriber-ag/source/src/Tools/CMakeLists.txt	2012-02-06 00:27:58.000000000 +0100
@@ -133,3 +133,9 @@
     ${APPLE_LINKS}
     ${WIN32_LINKS}
 )
+install(
+	TARGETS	add_section clean_tag convert_tag extract_speaker_segments merge_tag print_spk printtag tagstats testag
+	
+	RUNTIME	DESTINATION bin
+	LIBRARY	DESTINATION lib
+)
Index: transcriber-ag/CMakeLists.txt
===================================================================
--- transcriber-ag.orig/CMakeLists.txt	2012-02-06 00:27:56.000000000 +0100
+++ transcriber-ag/CMakeLists.txt	2012-02-06 00:56:52.000000000 +0100
@@ -1,5 +1,6 @@
 
 cmake_minimum_required	(VERSION 2.6)
+add_definitions(-DPREFIX=\"${CMAKE_INSTALL_PREFIX}\")
 add_subdirectory	(libs)
 add_subdirectory	(source)
 
Index: transcriber-ag/source/src/DataModel/DataModel_Cfg.cpp
===================================================================
--- transcriber-ag.orig/source/src/DataModel/DataModel_Cfg.cpp	2012-01-21 22:40:47.000000000 +0100
+++ transcriber-ag/source/src/DataModel/DataModel_Cfg.cpp	2012-02-06 00:51:50.000000000 +0100
@@ -80,18 +80,18 @@
 		if ( !Glib::file_test(m_localDTD, Glib::FILE_TEST_EXISTS) )
 		{
 			m_agOptions["system"] = "TransAG-3.0.dtd";
-			m_localDTD = "/usr/local/etc/TransAG/TransAG-3.0.dtd";
+			m_localDTD = PREFIX "/etc/TransAG/TransAG-3.0.dtd";
 
 			if ( !Glib::file_test(m_localDTD, Glib::FILE_TEST_EXISTS) )
 			{
 				m_agOptions["system"] = "TransAG-2.0.dtd";
-				m_localDTD = "/usr/local/etc/TransAG/TransAG-2.0.dtd";
+				m_localDTD = PREFIX "/etc/TransAG/TransAG-2.0.dtd";
 			}
 
 			if ( !Glib::file_test(m_localDTD, Glib::FILE_TEST_EXISTS) )
 			{
 				m_agOptions["system"] = "TransAG-1.0.dtd";
-				m_localDTD = "/usr/local/etc/TransAG/TransAG-1.0.dtd";
+				m_localDTD = PREFIX "/etc/TransAG/TransAG-1.0.dtd";
 			}
 		}
 
@@ -200,7 +200,7 @@
 
 		string pdir = curpath;
 		if ( pdir.empty() )
-			pdir="/usr/local";
+			pdir = PREFIX;
 		else {
 			if ( FileInfo(curpath).isFile() ) pdir = FileInfo(pdir).dirname();
 
@@ -229,8 +229,8 @@
 				else lib_path = "";
 			}
 
-			if ( lib_path.empty() && FileInfo("/usr/local/lib/ag").exists() ) {
-				lib_path = "/usr/local/lib:/usr/local/lib/ag";
+			if ( lib_path.empty() && FileInfo(PREFIX "/lib/ag").exists() ) {
+				lib_path = PREFIX "/lib:" PREFIX "/lib/ag";
 			}
 			if ( ! lib_path.empty() ) {
 				if ( !ld_library_path.empty() ) ld_library_path += ":";
@@ -266,7 +266,7 @@
 		if ( ! etc_ok )
 		{
 			// check if exists in in /usr/local/etc
-			string dir = "/usr/local/etc/TransAG";
+			string dir = PREFIX "/etc/TransAG";
 			if ( FileInfo(dir).exists() ) {
 				etc_ok = true;
 				configuration_path = dir ;
Index: transcriber-ag/source/src/Tools/txt2tag.cpp
===================================================================
--- transcriber-ag.orig/source/src/Tools/txt2tag.cpp	2012-01-21 22:40:47.000000000 +0100
+++ transcriber-ag/source/src/Tools/txt2tag.cpp	2012-02-06 00:55:59.000000000 +0100
@@ -130,8 +130,8 @@
 {
 	if (g_getenv("LD_LIBRARY_PATH") == NULL ) {
 		string lib_path("");
-		if ( FileInfo("/usr/local/lib/ag").exists() ) {
-			lib_path = "/usr/local/lib:/usr/local/lib/ag";
+		if ( FileInfo(PREFIX "/lib/ag").exists() ) {
+			lib_path = PREFIX "/lib:" PREFIX "/lib/ag";
 		} else {
 			gchar* cfgdir = g_path_get_dirname(progname);
 			if ( !g_path_is_absolute (cfgdir) ) {
